web-testbed/tests/tests_backend/proxies/non_widget_proxy.py

```diff
@@ -0,0 +1,17 @@
+from .base_proxy import BaseProxy
+
+
+class NonWidgetProxy(BaseProxy):
```

Comment:
There shouldn't be a need to differentiate Widgets and "non-widgets" - they're all just "objects" as far as Python is concerned.